### ISSUE TITLE
Minimal documentation about how to use the command line tools

### ISSUE THREAD

**Issue by fititnt** [EticaAI/HXL-Data-Science-file-formats#1]
TL;DR: This issue is about how to install the `bin/` scripts.

## How to install now

- The `hxl2example` (simply output CSV from direct from libhxl) and the hxl2tab at this moment are one-file-put-on-your-path scripts.
- As long as the dependencies are already installed they are likely to work.
  - Python 3 is an requirement
  - The python library "libhxl" (https://github.com/HXLStandard/libhxl-python; https://pypi.org/project/libhxl/) is an requeriment for every tool from this repository
  - Some additional tools may require other libraries. At bare minimum this will be documented per script

## Installation via pip
**At this moment, there is no way to install these scripts via pip.**

Maybe this will not be implemented at all, in special if the dependencies between different exporters get too complicated to keep it from breaking a system.

## How to create new exporters
Even if this set of tools get complicated over time, at least the [`bin/hxl2example`](https://github.com/EticaAI/HXL-Data-Science-file-formats/blob/main/bin/hxl2example) may be kept as minimum viable product to create some exporter that actually could be shared with others.

Also, without refactoring, other scripts like `hxl2tab` may still this way for long time. This may be useful, even if for testing, because some exporters may have to implement opinionated HXL attributes, and on worst case scenario if some the conversion table is not external (like remote spreadsheet or local HXLated csv) this can work for ad hoc changes.

**Comment by fititnt** [comment]
One way to install is using the Python pypi package HDP Toolchain at https://pypi.org/project/hdp-toolchain/.

Some scripts still have possibility to simply copy and paste the file to some some place on searchable path. Also each script already have extra inline documentation.
